<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMEmployeeSettingsWidget</class>
 <widget class="QWidget" name="QMEmployeeSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mitarbeiter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Daten</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cbHideDeactivated">
           <property name="text">
            <string>Deaktivierte Mitarbeiter ausblenden</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="leEmployeeFilter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbEmployeeFilter">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Funktionen Filtern</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../icons.qrc">
               <normaloff>:/icons/icons/view-filter.svg</normaloff>:/icons/icons/view-filter.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="tvEmployee">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QToolButton" name="tbAddEmployee">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Neuen Mitarbeiter hinzufügen</string>
               </property>
               <property name="text">
                <string>Hinzufügen</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/list-add-user.svg</normaloff>:/icons/icons/list-add-user.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbDeactivate">
               <property name="statusTip">
                <string>Deaktivieren des selektierten Mitarbeiters</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/list-remove-user.svg</normaloff>:/icons/icons/list-remove-user.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbEmployeeDetails">
               <property name="toolTip">
                <string>Details des aktuell selektierten Mitarbeiters anzeigen und bearbeiten</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/traindata.svg</normaloff>:/icons/icons/traindata.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbRevertChanges">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Alle Änderungen an Mitarbeiter und Gruppen zurücksetzen</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Gruppen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tvEmployeeGroups">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="tbEmployeeGroupsAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Hinzufügen einer neuen Funktionsgruppe</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../icons.qrc">
               <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbEmployeeGroupsRemove">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Entfernen einer selektierten Funktionsgruppe</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../icons.qrc">
               <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbAddEmployee</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>addEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbEmployeeFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>updateFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEmployeeFilter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>updateFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbRevertChanges</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>revertChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbEmployeeDetails</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>showEmployeeDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbEmployeeGroupsAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>addEmployeeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbEmployeeGroupsRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>removeEmployeeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbDeactivate</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>deactivate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbHideDeactivated</sender>
   <signal>clicked(bool)</signal>
   <receiver>QMEmployeeSettingsWidget</receiver>
   <slot>switchDeactivatedVisibility(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addEmployee()</slot>
  <slot>renameEmployee()</slot>
  <slot>updateFilter()</slot>
  <slot>revertChanges()</slot>
  <slot>showEmployeeDetails()</slot>
  <slot>addEmployeeGroup()</slot>
  <slot>removeEmployeeGroup()</slot>
  <slot>deactivate()</slot>
  <slot>switchDeactivatedVisibility(bool)</slot>
 </slots>
</ui>
